components/HtmlFragment.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+defineProps<{
+  html?: string
+}>()
+onMounted (() => {
```

Comment:
I think you need to do `import { onMounted } from 'vue'` - in radiolab it was working without the import, but I remember we got a warning about it somewhere.

Reply:
This should be auto imported by Nuxt and i haven't had any issues with it yet, so I'm going to leave it for now.

Actually, I've removed this component already from another refactor.